incubator/hnc/cmd/manager/main.go

```diff
@@ -93,6 +93,8 @@ func main() {
 	flag.IntVar(&webhookServerPort, "webhook-server-port", 443, "The port that the webhook server serves at.")
 	uaArg := arrayArg{val: &config.UnpropagatedAnnotations}
 	flag.Var(&uaArg, "unpropagated-annotation", "An annotation that, if present, will be stripped out of any propagated copies of an object. May be specified multiple times, with each instance specifying one annotation. See the user guide for more information.")
+	exnArg := mapArg{val: &config.ExcludedNamespaces}
```

Comment:
I feel like I'd rather just have a single arrayArg and then transfer the values to the config objects, e.g.:

```
config.UnpropagatedAnnotations = uaArg.Get()
for _, ex := range exnArg.Get() {
  config.ExcludedNamespaces[ex] = true
}
```

Seems more Go-like with less magic.